lv_conf_template.h

```diff
@@ -62,6 +62,9 @@
 #  define LV_MEM_CUSTOM_REALLOC realloc
 #endif     /*LV_MEM_CUSTOM*/
 
+/*Number of the memory buffer*/
```

Comment:
If it becomes public we should give a more detailed description.

For example:
```suggestion
/*Number of the intermediate memory buffer used during rendering and other internal processing mechanisms. 
 *You will see and error log message if there wasn't enough buffers. */
```

Reply:
Done.